Workbooks/PostgreSQL Flexible Server/Performance/Autovacuum Blockers/Autovacuum Blockers.workbook

```diff
@@ -1413,7 +1413,7 @@
                       {
                         "type": 1,
                         "content": {
-                          "json": "## Stats on long running transactions\r\n\r\n**Note: In the troubleshooting guide we consider any transaction which is running for more than {MIN_XACT_TIME_MS_THRESHOLD_FOR_LONG_XACTS} milliseconds as a long running transaction.**\r\n\r\nConsideration\r\n\r\n* Connection Duration **(blue)** - Duration of session or connection (collection_time-backend_start). You will notice higher value in case if you are using conneciton pool.\r\n\r\n* Transaction Duration **(green)** - Duration of transaction (collection_time-xact_start)\r\n\r\nLong running transactions is based out of pg_stat_activity view in postgresql. If you want to know more about these columns, please visit https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW\r\n ",
+                          "json": "#### Stats on long running transactions\r\n\r\n**Note: In this troubleshooting guide, we classify transactions with a duration of more than 7,200,000 milliseconds as long-running transactions.**\r\n\r\nConsideration:\r\n\r\n* Connection Duration **(blue)** - Duration of session or connection (collection_time-backend_start). Higher values may be observed if you are using a connection pool.\r\n\r\n* Transaction Duration **(green)** - Duration of transaction (collection_time-xact_start)\r\n\r\nLong-running transactions are based on the pg_stat_activity view in PostgreSQL. To learn more about these columns, please visit the PostgreSQL documentation at https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW.",
```

Comment:
again do not hardcode the value of MIN_XACT_TIME_MS_THRESHOLD_FOR_LONG_XACTS

Reply:
![image](https://user-images.githubusercontent.com/127991103/236272332-bba3b445-6176-4510-976c-7d00e83729c7.png)

Corrected